### DISCUSSION TITLE
ERROR ~ The AWS Access Key Id you provided does not exist in our records.

### DISCUSSION THREAD

**Discussion by AsanchezMD** [nextflow-io/nextflow#5034] [Q&A]
Hello, I am trying to run nextflow using conda.
This is the command I run 
nextflow run nf-core/cutandrun --input test1.csv --outdir ../analysis/test1 --genome GRCh38 -profile conda

nf-core/cutandrun v3.2.2-g6e1125d

however, I get this error when trying to run the pipeline:

ERROR ~ The AWS Access Key Id you provided does not exist in our records. (Service: Amazon S3; Status Code: 403; Error Code: InvalidAccessKeyId; Request ID: Y9KA66SVSH859WP9; S3 Extended Request ID: hGCxYbjDEbY813U41DS+qzcsrsX+1YEn+hcV4suE4rryzT+lDSOSs/fi9ZIO8DJeKtQyWE9NTCk=; Proxy: null)

Can you point me to what needs to be updated on ~/.aws/credentials ?

**Comment by GianlucaMattei** [comment]
Same problem here. 
Anyone found a solution?
